ADMIN-PORT.md

```diff
@@ -0,0 +1,44 @@
+---
+RFC: 3
+Status: Proposed
+---
+
+# ADMIN PORT RFC
+
+## Abstract
+
+In Valkey.conf, Add a new config parameter: "admin-port" to allow internal administrators in a company or special users from public internet to connect to the Valkey server to execute some special commands or load specfic modules.
```

Comment:
Would we add it to all of the debug configs: https://github.com/valkey-io/valkey/blob/unstable/valkey.conf#L130 ?

Reply:
yes, we could